src/sensitivity.jl

```diff
@@ -54,3 +54,28 @@ function plotUnitSensitivity(paramRange, result, realParam, i)
     plot!([realParam], seriestype = "vline", margin = 0.3cm, legend = :false)
     ylims!((1E2, 1E5))
 end
+
+""" This function calculates the simulated cell number for a pair of drugs in their exact concentration.  """
+function calc_cellNum(pDr1, pDr2, g0)
+    Dr1CellNum = predict(pDr1, g0, 96.0)
+    Dr2CellNum = predict(pDr2, g0, 96.0)
+    normNum = 1.0 .- [Dr1CellNum/g0, Dr2CellNum/g0]
+    combin = -((normNum[1] + normNum[2] - normNum[1] * normNum[2]) .- 1.0) * g0 
+    return combin
+end
```

Comment:
I wrote these functions based on only one pair of concentrations so that the answer (the difference) will be a scalar number.